### ISSUE TITLE
Mob Targeting Feature?

### ISSUE THREAD

**Issue by Atomic646** [EasyFarm/EasyFarm#303]
Can you add a feature to targeting with options like  "Mob HP must be 100%" otherwise keep looking?

**Comment by Mercurian** [comment]
No. Find your own god damn camp.

**Comment by MJohnHBerry** [comment]
And with that, Merry Christmas!

**Comment by Atomic646** [comment]
actually, its due to piece of shit MPK's being assholes. So before you open your cock holster, maybe you should get the entire story fuck nugget

**Comment by Mercurian** [comment]
We don't mpk each other just for giggles.

But if you show up in my camp, I'll MPK you every time. 

Respect is the number one rule for easy farming.

**Comment by Atomic646** [comment]
clearly your not on asura "land of the assholes"
who go out of there way to inconvenience people just because they can.

But it doesn't matter anymore, Iv rebuilt EF to a version that actually fully functions and has what I asked for.

**Comment by Mercurian** [comment]
I've been using EF for right around 6 years and never had anyone fuck with me even one time. 

If you're not an asshole, no one will bother you.

**Comment by Atomic646** [comment]
again, you obviously are not on asura. I like how you assume so much, not bothering anyone, not hoarding, or camping on anyone. in fact 99% im the only one in the zone. so go bark up a different tree.

but as I said before, It doesn't matter anymore, I cant be mpk'd now even if several tried.

**Comment by zaxtiss** [comment]
thats not true if all your using the is the mobs HP % all anyone would need
to do is use a no dmg type enfeeble right before you hit it and it will
mess you up.

**Comment by phoenixgold1980** [comment]
Will love to see the change he made . Want to do some add but know nothing to lua programming

**Comment by AzoRAI** [comment]
@Atomic646 If you've made reasonable updates, submit a PR.  Don't brag in the issues section for the software you're leeching.
